client/src/components/questions/Answer.jsx

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+import API_KEY from '../../../../config.js';
+
+const StyledAnswer = styled.div`
+  padding: 0.7% 0;
+`;
+
+
+const dateParser = (rawDate) => {
+  var date = new Date(rawDate.replace(/-/g, '/').replace(/T.+/, ''));
+  var dateStr = date.toDateString();
+
+  dateStr = dateStr.slice(dateStr.indexOf(' '));
+
+  var secondSpaceIndex = dateStr.indexOf(' ', 5);
+
+  dateStr = dateStr.slice(0, secondSpaceIndex) + ',' + dateStr.slice(secondSpaceIndex);
+
+  return dateStr;
+};
+
+const Answer = ({ answerData }) => {
+
+  const [reportText, setReportText] = useState(<u>Report</u>);
```

Comment:
I didn't realize you could put HTML in the state. That is great. no need to comment on this, you can just close this conversation out.